tests/python/pants_test/testutils/process_test_util.py

```diff
@@ -0,0 +1,18 @@
+# coding=utf-8
+# Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+import psutil
+
+
+def check_process_exists_by_command(name):
+  result = False
+  for proc in psutil.process_iter():
+    try:
+      result |= name in ''.join(proc.cmdline())
```

Comment:
it would probably make sense to short-circuit here and just `if name in ''.join(proc.cmdline()): return True` to avoid further iteration over subsequent processes after a true condition is first seen. then you could kill the `result = False` bit and just `return False` at the bottom also if no processes are found.

Reply:
Great point! I wasn't thinking! Thank you!